src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -38,11 +38,36 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.{BeforeAndAfterAll, PrivateMethodTester}
 import org.scalatestplus.mockito.MockitoSugar
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext.Implicits.{global => globalEc}
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.{Success, Try}
 
+class SamMockGoogleDirectoryDAO extends MockGoogleDirectoryDAO {
+  override def lockedDownGroupSettings = new Groups()
+    .setWhoCanAdd("ALL_OWNERS_CAN_ADD")
+    .setWhoCanJoin("INVITED_CAN_JOIN")
+    .setWhoCanViewMembership("ALL_MANAGERS_CAN_VIEW")
+    .setWhoCanViewGroup("ALL_OWNERS_CAN_VIEW")
+    .setWhoCanInvite("NONE_CAN_INVITE")
+    .setArchiveOnly(
+      "true"
+    ) // .setWhoCanPostMessage("NONE_CAN_POST") setting archive only is the way to set it so no one can post
```

Comment:
should `.setWhoCanPostMessage("NONE_CAN_POST")` be in this comment, or should it be on the next line like:
```suggestion
    ) // setting archive only is the way to set it so no one can post
    .setWhoCanPostMessage("NONE_CAN_POST")
```

Reply:
It should be in this comment - this is copied directly from the GoogleDirectoryDAO implementation file, without any changes.

Reply:
weird!